spec/models/sample_manifest/uploader_spec.rb

```diff
@@ -157,12 +156,24 @@
       download.save(test_file_name)
       Delayed::Worker.delay_jobs = false
       uploader = described_class.new(test_file, SampleManifestExcel.configuration, user, false)
-      uploader.run!
+      expect { uploader.run! }.to change(BroadcastEvent, :count).by(1)
       expect(uploader).to be_processed
-      expect(BroadcastEvent.count).to eq broadcast_events_count + 1
       expect(uploader.upload.sample_manifest).to be_completed
     end
 
+    it 'will generate sample accessions', accessioning_enabled: true do
+      download =
+        build(
+          :test_download_plates,
+          manifest_type: 'plate_full',
+          columns: SampleManifestExcel.configuration.columns.plate_full.dup,
+          study: create(:open_study, accession_number: 'acc')
+        )
+      download.save(test_file_name)
+      uploader = described_class.new(test_file, SampleManifestExcel.configuration, user, false)
+      expect { uploader.run! }.to change(Delayed::Job, :count).by(4)
```

Comment:
4 is the number of samples in the test file? If so maybe could make that explicit by grabbing it dynamically somehow, or assigning it to a variable with a name, or just a short comment?

Reply:
Good suggestion, had done that elsewhere but then moved the test and seemingly lost that.